helpers/config.py

```diff
@@ -1721,7 +1724,15 @@ def __questions_redis(self):
         - primary back end
         - single server installation
         """
-        if self.primary_backend or not self.multi_servers:
+        if self.multi_servers:
+            self.__dict['run_redis_containers'] = CLI.yes_no_question(
+                'Do you want to run the redis containers from this server?',
```

Comment:
> Redis

I write `redis` in purpose since their logo use a small `r`.  But all the documentation is using a capital `R`. 

Let's use `Redis` then ;-)